sysadmin_base/models/hr_employee.py

```diff
@@ -0,0 +1,11 @@
+###############################################################################
+# For copyright and license notices, see __manifest__.py file in root directory
+###############################################################################
+from odoo import fields, models
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    has_dialoga_access = fields.Boolean(string='Has Dialoga access')
```

Comment:
Los campos los escribimos de la siguiente manera:`
```suggestion
    has_dialoga_access = fields.Boolean(
        string='Has Dialoga access',
    )
```
Esto hace más fácil la lectura de código, y si a futuro añades un "help", GIT sólo pintará una línea de más porque se habrá añadido un cambio, por lo que también será más fácil de detectar qué tipo de cambio se ha realizado